model/flow/block.go

```diff
@@ -78,8 +78,8 @@ func (s BlockStatus) String() string {
 // therefore proves validity of the block. A certified block satisfies:
 // Block.View == QC.View and Block.BlockID == QC.BlockID
 type CertifiedBlock struct {
-	Block *Block
-	QC    *QuorumCertificate
+	Block        *Block
+	CertifyingQC *QuorumCertificate
```

Comment:
renamed this to `CertifyingQC` to avoid any confusion between the QC embedded in `Block` (i.e. `CertifiedBlock.Block.QC`, which is the QC for the _parent_) vs the `CertifyingQC` (which is the QC for the `CertifiedBlock` itself)